contracts/teller-network/Escrow.sol

```diff
@@ -218,327 +188,256 @@ contract Escrow is Pausable, MessageSigned, Fees, Arbitrable, RelayRecipient {
         uint _tradeAmount,
         uint8 _tradeType,
         uint _assetPrice
-    ) internal returns(uint escrowId) {
-        address seller;
-        address arbitrator;
+    ) internal {
         bool deleted;
 
-        (,,,,seller, arbitrator, deleted) = metadataStore.offer(_offerId);
+        (token, , , , seller, arbitrator, deleted) = metadataStore.offer(_offerId);
 
         require(!deleted, "Offer is not valid");
-        require(get_sender() == _buyer || get_sender() == seller, "Must participate in the trade");
```

Comment:
Why remove this check?

Reply:
Because now since the buyers are required to sign a message to create an escrow, they will always be a participant on the trade; and this signature is unique due to it containing an user nonce that increments on each trx, so there is no way of creating a escrow without the buyer participating in the trade